src/backend/storeManagers/nile/user.ts

```diff
@@ -0,0 +1,122 @@
+import { LogPrefix, logDebug, logError, logInfo } from 'backend/logger/logger'
+import {
+  createAbortController,
+  deleteAbortController
+} from 'backend/utils/aborthandler/aborthandler'
+import {
+  NileLoginData,
+  NileRegisterData,
+  NileUserData
+} from 'common/types/nile'
+import { runRunnerCommand } from './library'
+import { existsSync, readFileSync } from 'graceful-fs'
+import { nileUserData } from 'backend/constants'
+import { configStore } from './electronStores'
+import { clearCache } from 'backend/utils'
+
+export class NileUser {
```

Comment:
do we need to use a class here? we could just use module level exports for the functions since a module is already like a static class

Reply:
I honestly wont change this. I prefer to have the same way we have it in Heroic for Nile, Gogdl and legendary, its easier to maintain.